LEGAL

```diff
@@ -440,7 +468,7 @@ mentioned below.
 
   >>>
     A C-program for MT19937, with initialization improved 2002/2/10.::
-    Coded by Takuji Nishimura and Makoto Matsumoto.  
+    Coded by Takuji Nishimura and Makoto Matsumoto.
```

Comment:
This spaces are intentionally there, to break this line from the following lines.